### ISSUE TITLE
Can't read/write simio timer channel that has higher number

### ISSUE THREAD

**Issue by tgtakaoka** [dlbeer/mspdebug#66]
In the following session, the address of `TACCR2` should be `0x0176` and should have `0x07 0x08` values written. But both `md` and `simio info TA3` commands can't report values of `TACCR2`, or `mw` command can't write into `TACCR2` `TACCR0` and `TACCR1` are working as expected though.

```
$ mspdebug simio
...
(mspdebug) simio add timer TA3
Added new device "TA3" of type "timer".
(mspdebug) mw 0x0170 1 2 3 4 5 6 7 8
(mspdebug) md 0x0170 0x10
    00170: 01 02 03 04 05 06 00 00 00 00 00 00 00 00 ff ff |................|
(mspdebug) simio info TA3
Base address: 0x0160
IV address:   0x012e
IRQ0:         9
IRQ1:         8

TACTL:        0x0000
TAR:          0x0201
TAIV:         0x00

Channel  0, TACTL = 0x0000, TACCR = 0x0403
Channel  1, TACTL = 0x0000, TACCR = 0x0605
Channel  2, TACTL = 0x0000, TACCR = 0x0000
(mspdebug)
```

**Comment by tgtakaoka** [comment]
This issue should be fixed by pull request #62.

**Comment by tgtakaoka** [comment]
Fixed.
